Gems/Multiplayer/Code/Include/Multiplayer/Components/NetworkDebugPlayerIdComponent.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <Source/AutoGen/NetworkDebugPlayerIdComponent.AutoComponent.h>
+#include <Atom/RPI.Public/ViewportContext.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzFramework/Font/FontInterface.h>
+
+namespace AZ::RPI
+{
+    class ViewportContext;
+}
+
+namespace Multiplayer
+{
+    class NetworkDebugPlayerIdComponent
```

Comment:
Can we have a public class description that talks to intent/responsibilities?

Reply:
Good call